package.json

```diff
@@ -18,7 +19,8 @@
     "validator": "^12.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "dev": "react-scripts start",
```

Comment:
Do we have controll over webpack and other build stuff?

Reply:
This will need to be changed when we eject cra, react-scripts is the "cra" way of starting the development server.